Source/Bridge/UserAgent.swift

```diff
@@ -1,8 +1,15 @@
 import Foundation
 
-public enum UserAgent {
-    public static func userAgentSubstring(for componentTypes: [BridgeComponent.Type]) -> String {
+enum UserAgent {
+    static func build(componentTypes: [BridgeComponent.Type], applicationPrefix: String?) -> String {
```

Comment:
I'm a bit worried about this change since it was previously public. But I don't think we need/want it to be anymore. Since we are bumping to v1.1 I think that should be OK - I like hiding this from developers.

Reply:
I'm not worried about this, since we're already breaking the API and there's no application need to build the user agent manually. Can we flip the param order since the `applicationPrefix` will be first in the user agent?

Reply:
Done!